<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1121</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 151, 209);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter Matriel</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>0</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/387499949_851957709935185_8033033763751298194_n.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>561</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(183, 214, 239);</string>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="reference">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>REFERENCE :</string>
       </property>
      </widget>
      <widget class="QLabel" name="etat">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ETAT :</string>
       </property>
      </widget>
      <widget class="QLabel" name="quantite">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>QUANTITE :</string>
       </property>
      </widget>
      <widget class="QLabel" name="nom">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Line_Edit_referance">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Line_Edit_nom">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Line_Edit_quantite">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_ajouter">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>130</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>AJOUTER</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_supprimer">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>190</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>SUPPRIMER</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_modifier">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>50</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>MODIFIER</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_etat">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>excellent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>en panne</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>300</y>
        <width>581</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(183, 214, 239);</string>
      </property>
      <property name="title">
       <string>affichage</string>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>551</width>
         <height>291</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
